src/components/thumbnail-gallery/ThumbnailGallery.tsx

```diff
@@ -183,6 +179,15 @@ const ThumbnailGallery: React.FC<IThumbnailGalleryProps> = ({
                   }}
                 />
                 <div
+                  style={borderRadius !== 0 && titlePosition !== TitlePosition.BELOW ? {
+                    width: getWidth + 2 * padding + 'px',
+                    height: getHeight + 2 * padding + 'px',
+                    borderRadius: borderRadius + '%',
+                    position: "absolute",
```

Comment:
the values that do not include any variable can be defined inside the CSS file with className